ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
```

Comment:
Yes, a reminder of distinctions between SO, mailing list and issues would be valuable.